pkg/updater/updater.go

```diff
@@ -880,11 +884,11 @@ func alertRow(cols []*statepb.Column, row *statepb.Row, failuresToOpen, passesTo
 		latestID = row.CellIds[latestFailIdx]
 	}
 	msg := row.Messages[latestFailIdx]
-	return alertInfo(totalFailures, msg, id, latestID, firstFail, latestFail, latestPass)
+	return alertInfo(totalFailures, msg, id, latestID, firstFail, latestFail, latestPass, latestColumn)
```

Comment:
In our case the idea is to send emails until it's succeeded. 
If i'll use firstFail I'll keep sending emails to the user, right? 
Or because it's part of the "old_dasboard" we will not resend emails about it.

Reply:
Using firstFail vs. latestColumn doesn't actually matter for how often the mails get sent, that's controlled by user configuration. (It should only send an email when new failing rows are noticed, _or_ periodically until the tab passes if the user's configured it to do so).

Given that, I think firstFail might be more appropriate here; it matches with the 'start' of the failure.

Reply:
But then it's not supposed to be latestFail and not firstFail? 
Because we want to alert about the latest failure happens